### ISSUE TITLE
Use outside of Sprockets

### ISSUE THREAD

**Issue by jarednorman** [hauleth/vanilla-ujs#31]
Versions after `0.1.0` don't seem to be able to be (reasonably) used outside of Sprockets (within other build tools, like Webpack, for example). Currently installing this package from npm seem kind of pointless since it contains Sprockets directives that don't work outside of Sprockets.

Was this a conscious decision? With Rails moving towards being friendlier to external asset tools with Webpacker and all that, it seems like this project should support that stuff. Thoughts?

**Comment by hauleth** [comment]
I can check this, however I was hinking about moving to ES2015 and using imports instead of Sprockets.

**Comment by prem-prakash** [comment]
tried to use as node package + webpack import

```js
import 'vanilla-ujs/lib/assets/javascripts/vanilla-ujs/polyfills.js';
import 'vanilla-ujs/lib/assets/javascripts/vanilla-ujs/confirm.js';
import 'vanilla-ujs/lib/assets/javascripts/vanilla-ujs/csrf.js';
import 'vanilla-ujs/lib/assets/javascripts/vanilla-ujs/disable.js';
import 'vanilla-ujs/lib/assets/javascripts/vanilla-ujs/form.js';
import 'vanilla-ujs/lib/assets/javascripts/vanilla-ujs/liteajax.js';
import 'vanilla-ujs/lib/assets/javascripts/vanilla-ujs/method.js';
```

but it does not work

it says matches as not defined

**Comment by jarednorman** [comment]
Now that rails-ujs exists with no dep on jQuery, I just use that. I'm unfollowing this issue, but not closing it since other people seem to want this feature. :heart:

**Comment by quentindemetz** [comment]
I managed to use this with with webpacker using the following code:

```
 var matches = (function(doc) {
   return doc.matchesSelector ||
     doc.webkitMatchesSelector ||
     doc.mozMatchesSelector ||
     doc.oMatchesSelector ||
     doc.msMatchesSelector;
 })(document.documentElement);

import 'vanilla-ujs/lib/assets/javascripts/vanilla-ujs/polyfills.js';
import 'vanilla-ujs/lib/assets/javascripts/vanilla-ujs/confirm.js';
import 'vanilla-ujs/lib/assets/javascripts/vanilla-ujs/csrf.js';
import 'vanilla-ujs/lib/assets/javascripts/vanilla-ujs/disable.js';
import 'vanilla-ujs/lib/assets/javascripts/vanilla-ujs/form.js';
import 'vanilla-ujs/lib/assets/javascripts/vanilla-ujs/liteajax.js';
import 'vanilla-ujs/lib/assets/javascripts/vanilla-ujs/method.js';
```

**Comment by AlfonsoUceda** [comment]
I'm going to take advantage of this issue and ask if there is a chance to remove railsties gem and make this gem agnostic.
